src/renderer/components/workspaces/PanelLayout.tsx

```
@@ -609,11 +591,7 @@ function SplitResizer({
       : { top: `${ratio * 100}%` };
 
   return (
-    <div
-      className={resizerClass}
-      style={style}
-      onMouseDown={handleMouseDown}
-    />
+    <div className={resizerClass} style={style} onMouseDown={handleMouseDown} />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)